tests/run/tryfinally.pyx

```diff
@@ -460,6 +460,7 @@ def complex_finally_clause(x, obj):
     Traceback (most recent call last):
     TypeError
     """
+    a = 0
```

Comment:
I'm not sure I understand what this is doing

Reply:
The C compiler cannot see through Python context managers, so without this pre-initialisation, it may issue a warning about uninitialised usage. I'll push the change separately.